numpy/_typing/_add_docstring.py

```diff
@@ -149,4 +149,45 @@ def _parse_docstrings() -> str:
 
     """)
 
+add_newdoc('Array', repr(Array),
+    """
+    A `np.ndarray[tuple[*Ts], np.dtype[+ScalarType]] <numpy.ndarray>` type
+    alias :term:`generic <generic type>` w.r.t. its shape and
+    `dtype.type <numpy.dtype.type>`.
+
+    Can be used during runtime for typing arrays with a given dtype
```

Comment:
I'd drop the "during runtime" part, as the statement is equally valid both in .py and .pyi files